cypress_shared/helpers/index.ts

```diff
@@ -47,7 +47,16 @@ const riskToSelfSummariesFromApplication = (
 }
 
 const tableRowsToArrays = (tableRows: Array<TableRow>): Array<Array<string>> => {
-  return tableRows.map(row => Object.keys(row).map(i => (row[i].html ? Cypress.$(row[i].html).text() : row[i].text)))
+  return tableRows.map(row => Object.keys(row).map(i => uiObjectValue(row[i])))
+}
+
+const uiObjectValue = (tableCell: TableCell) => {
+  const tableCellHtml = tableCell as HtmlItem
+  if (typeof tableCellHtml.html === 'string') {
```

Comment:
Would `if ('html' in item) {` be better here than casting?

Reply:
Yes